src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/BlockParentHandler.java

```diff
@@ -151,13 +151,26 @@ protected DetailAST getRightCurly() {
      * Check the indentation of the left curly brace.
      */
     protected void checkLeftCurly() {
-        // the lcurly can either be at the correct indentation, or nested
-        // with a previous expression
-        final DetailAST lcurly = getLeftCurly();
-        final int lcurlyPos = expandedTabsColumnNo(lcurly);
+        checkCurly("lcurly", getLeftCurly());
```

Comment:
I assume everything in this file is the same, just a code simplification change?